anime-list-master.xml

```diff
@@ -38728,6 +38735,14 @@
   </anime>
   <anime anidbid="14086" tvdbid="347734" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Super Dragon Ball Heroes</name>
+	<mapping-list>
+      <mapping anidbseason="0" tvdbseason="0"/>
+      <mapping anidbseason="1" tvdbseason="1" start="1" end="6"/>
+      <mapping anidbseason="1" tvdbseason="2" start="7" end="19" offset="-6"/>
+      <mapping anidbseason="1" tvdbseason="3" start="20" end="30" offset="-19"/>
+      <mapping anidbseason="1" tvdbseason="4" start="31" end="38" offset="-30"/>
+      <mapping anidbseason="1" tvdbseason="5" start="39" end="48" offset="-38"/>
+    </mapping-list>
   </anime>
```

Comment:
- Might as well take the chance to change the default to absolute. That will let parser that support it know that the TVDB mapping is correct.
- Opening is over-indented. The list uses spaces, not tabs.
- Season 0>0 is implied

```suggestion
  <anime anidbid="14086" tvdbid="347734" defaulttvdbseason="a" episodeoffset="" tmdbid="" imdbid="">
    <name>Super Dragon Ball Heroes</name>
    <mapping-list>
      <mapping anidbseason="1" tvdbseason="2" start="7" end="19" offset="-6"/>
      <mapping anidbseason="1" tvdbseason="3" start="20" end="30" offset="-19"/>
      <mapping anidbseason="1" tvdbseason="4" start="31" end="38" offset="-30"/>
      <mapping anidbseason="1" tvdbseason="5" start="39" end="48" offset="-38"/>
    </mapping-list>
  </anime>
```